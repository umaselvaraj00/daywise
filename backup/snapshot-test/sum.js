const sum=(a,b,c)=>{
    return a+b+c;
}
export default sum;