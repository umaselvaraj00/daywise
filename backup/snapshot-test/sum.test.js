import sum from "./sum"
// describe ("test suite 1", ()=>{


test("snapshot testing - sum",()=>{
    expect(sum(3,4,2)).toMatchSnapshot();
    expect(sum(4,4,0)).toMatchSnapshot();
    expect(sum(-3,4,3)).toMatchSnapshot();
    expect(sum(3,5,7)).toMatchSnapshot();
    expect(sum(3,4,2)).toMatchSnapshot();
});
