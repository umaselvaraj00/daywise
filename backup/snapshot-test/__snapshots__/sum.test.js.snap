// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing - sum 1`] = `9`;

exports[`snapshot testing - sum 2`] = `8`;

exports[`snapshot testing - sum 3`] = `4`;

exports[`snapshot testing - sum 4`] = `15`;

exports[`snapshot testing - sum 5`] = `9`;
